selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AdminOfParentOrganizationViewsTest.java

```diff
@@ -107,17 +100,20 @@ public void testOrganizationListComponents() {
     assertTrue(headers.contains(AVAILABLE_RAM.getTitle()));
     assertTrue(headers.contains(SUB_ORGANIZATIONS.getTitle()));
     assertTrue(headers.contains(ACTIONS.getTitle()));
-
     assertTrue(
         organizationListPage.getValues(NAME).contains(parentOrganization.getQualifiedName()));
     assertTrue(organizationListPage.getValues(NAME).contains(childOrganization.getQualifiedName()));
-    assertFalse(organizationListPage.getValues(NAME).contains(rootOrganization.getQualifiedName()));
   }
 
-  @Test(priority = 2)
-  public void testParentOrganization() {
-    organizationListPage.clickOnOrganization(parentOrganization.getName());
+  @Test
+  public void parentOrganizationViewsTest() {
```

Comment:
According to the test name convention, **test** word goes to the start of _method name_ to the contrary to _test class name_ where **test** word goes to the end of name.

Reply:
Also **"views"** looks as redundant here.